config/routes.rb

```diff
@@ -178,16 +178,30 @@ def self.matches?(request)
       end
     end
 
-    # TODO: investigate whether this could live in the
-    # resources :digitization_queue_items > member do block.
-    # Something like delete :delete_comment
-    # but with an additional :comment_id parameter.
-    # For now, just do the obvious:
     delete  "digitization_queue_items/:id/delete_comment/:comment_id",
       to: "digitization_queue_items#delete_comment",
       as: "delete_comment"
 
 
+    # START R&R
+
+    # GET /admin/r_and_r_items/
+    get "/r_and_r_items",    to: "r_and_r_items#index"
+    # GET /admin/r_and_r_item/4
+    get "/r_and_r_item/:id", to: "r_and_r_items#show", as: "r_and_r_item"
+    # GET /admin/r_and_r_items/1/edit
+    get "/r_and_r_item/:id/edit", to: "r_and_r_items#edit", as: "edit_r_and_r_item"
+    # PATCH /admin/r_and_r_items/1/update
+    patch "/r_and_r_item/:id", to: "r_and_r_items#update", as: "update_r_and_r_item"
+    # GET /admin/r_and_r_items/new
+    get "/r_and_r_items/new", to: "r_and_r_items#new",  as: "new_r_and_r_item"
+    # POST  /admin/r_and_r_items/
+    post "/r_and_r_items", to: "r_and_r_items#create", as: "create_r_and_r_item"
+    # DELETE /admin/r_and_r_items/1
+    delete "/r_and_r_item/:id", to: "r_and_r_items#destroy", as: "delete_r_and_r_item"
+
+    # END R&R
```

Comment:
Instead of listing each route individually, it might be better to use the Rails routing `resources` method for standard routes. Compare to what's being done for `resources :digitization_queue_items`. That might help with the issues with having to override simple_form_for for non-standard routing.

Reply:
Done.